src/sass/layouts/_header.scss

```diff
@@ -0,0 +1,212 @@
+.header {
+  backdrop-filter: blur(1rem);
+  display: flex;
+  position: fixed;
+  z-index: 100;
+  max-height: 8rem;
+  width: 100%;
+  padding: 2rem 4rem;
+  justify-content: space-between;
+  align-items: center;
+  background-image: linear-gradient(
+    to right,
+    rgb(55 65 81 / 30%),
+    rgb(55 65 81 / 60%)
+  );
+
+  @include respond(tab-port) {
+    padding: 2rem 3rem;
+  }
+
+  @include respond(phone) {
+    padding: 2rem;
+  }
+
+  &__logo-container {
+    flex: 0 0 5%;
+    min-width: 5.5rem;
+
+    &__img {
+      height: 100%;
+      max-width: 100%;
+    }
+  }
+
+  &__btn-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+
+    @include respond(phone) {
+      flex-direction: column;
+      justify-content: center;
+      height: 100vh;
+      width: 60%;
+      position: absolute;
+      left: 100%;
+      top: 0;
+      z-index: 101;
+      background-image: linear-gradient(to right, rgb(55 65 81), rgb(55 65 81));
+      transition: all 0.5s;
+      visibility: hidden;
+      opacity: 0;
+    }
+
+    &__btn {
+      font-family: var(--font-family);
+      text-decoration: none;
+      padding: 1rem 2rem;
+      margin-left: 2rem;
+      background-color: inherit;
+      font-size: 1.9rem;
```

Comment:
you can create some css variables for font sizes too.

https://tailwindcss.com/docs/font-size

Don't copy all from tailwind, only add whatever is useful for you app